modules/ossm-rn-new-features.adoc

```diff
@@ -15,6 +15,179 @@ Module included in the following assemblies:
+//also included in "Upgrading --> Upgrading 2.5 to 2.6" but added here for increased visibility.
+//Dean
+This release disables {JaegerName} by default for new instances of the `ServiceMeshControlPlane` resource.
+
+When updating existing instances of the `ServiceMeshControlPlane` resource to {SMProductName} version 2.6, {JaegerShortName} remains enabled by default.
+
+{SMProductName} 2.6 is the last release that includes support for {JaegerName} and {es-op}. Both {JaegerShortName} and {es-op} will be removed in the next release. If you are currently using {JaegerShortName} and {es-op}, you need to switch to {TempoName} and {OTELName}.
+
+//Gateway API Update for 2.6 OSSM-5854
+//Kubernetes Gateway API and {product-title} Gateway API are the same. It is referenced as {product-title} Gateway API in 2.5 and as {product-title} Gateway API here https://docs.openshift.com/container-platform/4.15/nodes/clusters/nodes-cluster-enabling-features.html so to be consistent, it is also referenced as {product-title} Gateway API for 2.6.
+=== {product-title} Gateway API generally available for cluster-wide deployments
+//Jacek
+This release introduces the General Availability of {product-title} Gateway API, also known as the Kubernetes Gateway API, that is enabled by default only for cluster-wide deployments.
+
+Gateway API is now enabled by default if cluster-wide mode is enabled (`spec.mode: ClusterWide`). It can be enabled even if the custom resource definitions (CRDs) are not installed in the cluster.
+
+[IMPORTANT]
+====
+Gateway API for multitenant mesh deployments is still in Technology Preview.
+====
+
+Refer to the following table to determine which Gateway API version should be installed with the OpenShift {SMProductShortName} version you are using:
+
+|===
+|Service Mesh Version | Istio Version | Gateway API Version | Notes
+
+|2.6
+|1.20.x
+|1.0.0
+|N/A
+
+|2.5.x
+|1.18.x
+|0.6.2
+|Use the experimental branch because `ReferenceGrand` is missing in v0.6.2
+
+|2.4.x
+|1.16.x
+|0.5.1
+|For multitenant mesh deployment, all Gateway API CRDs must be present. Use the experimental branch.
+|===
+
+You can disable this feature by setting `PILOT_ENABLE_GATEWAY_API` to `false`:
+
+[source,yaml]
+----
+spec:
+  runtime:
+    components:
+      pilot:
+        container:
+          env:
+            PILOT_ENABLE_GATEWAY_API: "false"
+----
+
+//Please note that the support for Kubernetes Gateway API can be enabled even if the CRDs are not installed in the cluster.
+
+[id="new-features-ossm-2-5-3"]
+== New Features {SMProductName} version 2.5.3
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs), contains bug fixes, and is supported on {product-title} 4.12 and later.
+
+=== Component versions for {SMProductName} version 2.5.3
+
+// Release is scheduled for July 10, 2024.
+//Includes 2.6, 2.5.3, 2.4.z. 2.6 ends support for v2.3
+
+|===
+|Component |Version
+
+|Istio
+|1.18.5
+
+|Envoy Proxy
+|1.26.8
+
+|Kiali
+|1.73.8
```

Comment:
I think this should be 1.73.9?

Reply:
There are no component version listed here yet: https://issues.redhat.com/browse/OSSM-6435 so the Component Tables have not been updated.